common/types/src/main/java/io/helidon/common/types/Annotation.java

```diff
@@ -40,8 +67,8 @@ public interface Annotation extends AnnotationBlueprint, Prototype.Api, Comparab
      *
      * @return a new builder
      */
-    static Annotation.Builder builder() {
-        return new Annotation.Builder();
+    static Builder builder() {
```

Comment:
is this what IDE did? Since I think for merging purposes it would be better to leave it with `Annotation` prefix.

Reply:
yes, will restore original approach